CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Unreleased
+
+* Breaking: Allow values larger than 32 bit for `int!` and `uint!` macros.
+  This means that explicit `_i32` and `_u32` literals are no longer valid and it also raises the MSRV to 1.79.
```

Comment:
```suggestion
  This means that `i32`-suffixed and `u32`-suffixed literals are no longer accepted
```

Can you make the MSRV thing a separate bullet point?

Reply:
Actually, I wonder.. Since the `const` block is in a macro, shouldn't the MSRV be unaffected? It's still a breaking change, but the crate can still be built by older versions of rustc, right?

Reply:
Looks too be 1.46: https://releases.rs/docs/1.46.0/
Let's use that if possible. js_int is a basic enough crate that I would prefer a lower MSRV.